docs/clients.mdx

```diff
@@ -35,7 +35,7 @@ This page provides an overview of applications that support the Model Context Pr
 | [Goose][Goose]                                   | ❌          | ❌        | ✅      | ❓                     | ❌         | ❌    | Supports tools.                                                                                 |
 | [gptme][gptme]                                   | ❌          | ❌        | ✅      | ❓                     | ❌         | ❌    | Supports tools.                                                                                 |
 | [HyperAgent][HyperAgent]                         | ❌          | ❌        | ✅      | ❓                     | ❌         | ❌    | Supports tools.                                                                                 |
-| [JetBrains AI Assistant][JetBrains AI Assistant] | ❌          | ❌        | ✅      | ❌                     | ❌         | ❌    | Supports tools for all JetBrains IDEs.                                                                                 |
```

Comment:
This change is unrelated, currently the rendering is fine it seems on the actual page https://modelcontextprotocol.io/clients - if it needs fixing, it's best to open a separate PR.